includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -92,7 +93,38 @@ function get_columns() {
 		$columns = apply_filters( 'sensei_analysis_overview_columns', $columns, $this );
 		return $columns;
 	}
-
+	/**
+	 * Append the count value to column headers where applicable
+	 *
+	 * @since  4.2.0
+	 * @access private
+	 *
+	 * @param array $columns Array of columns for the report table.
+	 * @return array The array of columns to use with the table with columns appended to their title
+	 */
+	public function add_totals_to_report_column_headers( array $columns ) {
+		remove_filter( 'sensei_analysis_overview_columns', array( $this, __FUNCTION__ ) );
+		$column_value_map = array();
+		switch ( $this->type ) {
+			case 'lessons' && $this->get_course_filter_value():
+				$total_counts                           = $this->get_totals_for_lesson_report_column_headers( $this->get_course_filter_value() );
+				$column_value_map['title']              = $total_counts->lesson_count;
+				$column_value_map['lesson_module']      = $total_counts->unique_module_count;
+				$column_value_map['students']           = $total_counts->unique_student_count;
+				$column_value_map['completions']        = $total_counts->lesson_completed_count > 0 ? $total_counts->lesson_completed_count : 0;
+				$column_value_map['days_to_completion'] = $total_counts->lesson_completed_count > 0 ? ceil( $total_counts->days_to_complete_sum / $total_counts->lesson_completed_count ) : __( 'N/A', 'sensei-lms' );
+				$column_value_map['completion_rate']    = $total_counts->lesson_start_count > 0 ? Sensei_Utils::quotient_as_absolute_rounded_percentage( $total_counts->lesson_completed_count, $total_counts->lesson_start_count ) . '%' : '0%';
```

Comment:
In case the expression is long (and I can't see it on my screen), I'd like to extract it into a separate line by assigning to a variable.

Reply:
Broke them into multiple lines here, please take a look if possible, thanks https://github.com/Automattic/sensei/pull/4911/commits/31682dfa53e81535cdd8290cebd9c22ebccced36